Library/Formula/infer.rb

```diff
@@ -0,0 +1,60 @@
+class Infer < Formula
+  desc "Facebook Infer is a static analysis tool - if you give Infer some Objective-C, Java, or C code, it produces a list of potential bugs."
+  homepage "https://fbinfer.com/"
+  url "https://github.com/facebook/infer/releases/download/v0.1.0/infer-osx-v0.1.0.tar.xz"
```

Comment:
Working on getting that changed over, but it'll likely be a good minute before I've got it. As it stands, there might be issues related to how opam and Infer's Makefiles expect your environment to be configured. Ideally, I'll need to find a way to avoid modifying to user's `.ocamlinit` file while not breaking ocamlfind.